src/allotropy/parsers/agilent_gen5/agilent_gen5_structure.py

```diff
@@ -620,6 +634,7 @@ def create_results(
         )
         for well_position, well_calculated_data in calculated_data.items()
         for label, value in well_calculated_data
+        if value != NEGATIVE_ZERO
```

Comment:
I noticed that in the output files, we are dropping values that are `0`

This is because (unfortunately for us) in python `0 == -0`.

I think the way to solve this is to leave `well_value` as None and do `if value is not None` here, and then above where you set `value=value`, change it to `value=value if value is not None else NEGATIVE_ZERO` (note you will also have to check for `None` in `_create_measurement`)

Reply:
It was really hard to follow this, but I think I got the idea. I managed to avoid dropping values that are 0 by comparing to None

Reply:
Sorry, but you got it!